### DISCUSSION TITLE
The login page doesn't refresh automatically to admin panel when running through Traefik Hub reverse proxy

### DISCUSSION THREAD

**Discussion by as-shura** [nextcloud/all-in-one#2555] [Questions] [answered]
### Steps to reproduce
1. Install Docker
2. Go to https://hub.traefik.io/ and create an account and install Traefik with agent for the Hub.
3. Log into the Hub and add a new agent
4. Use the generated code for the new agent and run the docker-compose
5. Identify the name of the network that docker create for the agent because this name has to be used in all the services that we want to be able to access through the Traefik Hub admin panel.
6. Add following for the NextCloud service
      networks:
        - traefik-hub_default
        
  traefik-hub_default:
    external: true
8. Create a NextCloud instance with docker-compose and link the service's network with the same network name as the Traefik Hub agent as in step 6 (there is no step 7).
9. Go in the Traefik Hub control panel and wait for the Nextcloud service to appear and then publish it.
10. Put the following NextCloud setting in environment variables : - NEXTCLOUD_TRUSTED_DOMAINS=(auto generated url from Traefik Hub for the published NextCloud service) and restart container.
11. When the service is now accessible log into NextCloud by using the url generated from Traefik Hub

### Expected behavior 

The login panel should refresh automatically but it stays idle even if the login is successful.

### Actual behavior 

We have to refresh the page manually to enter into admin panel.

### Host OS 

diepi latest version

#### Nextcloud AIO version 

NextCloud latest version

**Comment by szaimen** [comment, marked as answer]
Hi, can you follow https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md?
